templates/zerver/help/configure-email-notifications.md

```diff
@@ -0,0 +1,18 @@
+# Configure email notifications
+
+{!follow-steps.md!} configure your settings to receive email notifications
+for private messages and @-mentions sent while you are offline.
+
+{!go-to-the.md!} [Notifications](/#settings/notifications)
+{!settings.md!}
+
+2. Select the **Email notifications** option under the
+**Private messages and @-mentions** section.
+
+    !!! tip ""
+        If you want to receive **email digests**, emails that combine all of
+        your notifications into an email, instead of receiving one email for
+        every single notification, select the **Digest emails when I'm away**
+        option under the **Other notifications I want** section.
```

Comment:
The digest feature is actually basically unrelated to the rest -- it sends summaries of everything, not just PMs and mentions.  We might want to make that it's own doc?